srcpkgs/mdocml/files/cgi.h

```diff
@@ -0,0 +1,5 @@
+#define SCRIPT_NAME ""
+#define MAN_DIR "/home/void-man/man"
```

Comment:
I'd much rather this be in /var/lib/man-cgi/

Reply:
Changed.